### ISSUE TITLE
Cron trigger support for step values greater than the maximum

### ISSUE THREAD

**Issue by mdz** [agronholm/apscheduler#509]
**Is your feature request related to a problem? Please describe.**
I recently migrated a workload from Vixie cron to APScheduler, and discovered that the cron trigger only supports a subset of the Vixie crontab format.

Specifically, we had a couple of jobs configured with `*/60 * * * *` to run every 60 minutes. I assume this is equivalent to `0 * * * *`. cron accepted this, but APScheduler [https://github.com/agronholm/apscheduler/blob/d10f20215d8c78e9e2d32d634f276bb89f86ca38/apscheduler/triggers/cron/expressions.py#L34](rejects) it.

That particular example isn't that useful, but it turns out that things like `*/90 * * * *` are also allowed.

**Describe the solution you'd like**
It would be nice to support this in APScheduler for greater compatibility with cron.

**Comment by rafalkrupinski** [comment]
I don't know about Vixie, but AFAIK `/N` actually means values divisible by N. since there are no minutes in any hour divisible by 90, there's no need for such pattern.

**Comment by mdz** [comment]
*/90 runs the job every 90 minutes which is useful.

**Comment by rafalkrupinski** [comment]
OK, it's not mentioned in the vixie crontab man page, and both wikipidia and chatgpt say otherwise, but they are both often wrong.

Since aps is stateful, it makes perfect sense to support it.

**Comment by mdz** [comment]
Indeed. I discovered this behavior when I migrated a production workload from vixie cron to apscheduler, keeping the crontab entries intact. One of them failed because it relied on this (apparently undocumented) behavior

**Comment by agronholm** [comment]
This is not trivial to implement, but if somebody comes up with a PR that includes updated tests and documentation, I'm willing to consider it.
